src/layout/StepsAccordion.js

```diff
@@ -9,6 +10,7 @@ import { withTheme } from '../utils/theme'
 // We don't observe on Step because then it would rerender its children when `node`
 // changes, which unfocuses query inputs as soon as the first character is entered.
 let Buttons = _.flow(
+  setDisplayName('Buttons'),
   observer,
   withTheme
```

Comment:
I'd actually say don't compose `observer` with anything, since it's basically orthogonal with everything else.

For `setDisplayName`, I think it's more effort than it's worth. The endgame of that is making a composition with it for every HOC